src/main/java/racingcar/common/Constant.java

```diff
@@ -0,0 +1,6 @@
+package racingcar.common;
+
+public class Constant {
+    public final static String BLANK = "";
+    public final static String BLANK_STRING = " ";
```

Comment:
```suggestion
    public static final String BLANK_STRING = " ";
```
접근 제어자의 선언 순서를 지켜주세요 . 
https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers